content/en/observability_pipelines/_index.md

```diff
@@ -13,7 +13,7 @@ further_reading:
   text: "Sensitive Data Scanner"
 - link: "/agent/configuration/dual-shipping/#yaml-configuration"
```

Comment:
Just double-checking that you still mean for this to link to the yaml-configuration section specifically... I found that a bit odd

Reply:
Thanks for checking, yes it should go to the YAML configuration because that's how you configure dual shipping in the Agent.